### ISSUE TITLE
Can we get a new release?

### ISSUE THREAD

**Issue by TheJaredWilcurt** [tonyganch/gonzales-pe#302]
This was merged 5 days after the last release, 11 months ago:

* https://github.com/tonyganch/gonzales-pe/pull/295

**Comment by RajaBellebon** [comment]
Hello, not sure if it is right issue to ask sorry @TheJaredWilcurt but when I get the latest version of the package in our project, I am getting 

```
gonzales-pe@^4.2.4:
  version "4.2.4"
  resolved "https://packages.atlassian.com/api/npm/npm-remote/gonzales-pe/-/gonzales-pe-4.2.4.tgz#356ae36a312c46fe0f1026dd6cb539039f8500d2"
  integrity sha1-NWrjajEsRv4PECbdbLU5A5+FANI=
  dependencies:
    minimist "1.1.x"
```

But in the `package.json` , it is using ` minimist "^1.2.0"`?

I try to run `npm install` locally on `dev` and it seems that the one in the repository has not been updated.

Maybe I am missing something...

Sorry for the issue highjack and happy to raise a proper one for this case.

**Comment by hugovk** [comment]
@RajaBellebon: It's the same issue, and will be fixed by a release.

The dev branch has ^1.2.0:

https://github.com/tonyganch/gonzales-pe/blob/cb4be1c6c22184b525c6b7252823de834235a963/package.json#L32

But the last release, v4.2.4 has 1.1.x:

https://github.com/tonyganch/gonzales-pe/blob/4c03cf9b3945ea1dd2486fa13ddf4b2d5450f3f0/package.json#L32

**Comment by XhmikosR** [comment]
@jdalton @TheJaredWilcurt any chance you could get a new release out please? :)

EDIT: preferably after #304 is merged too so that everyone gets the fix without the need to do `npm audit fix` or similar.

**Comment by Berkmann18** [comment]
> @jdalton @TheJaredWilcurt any chance you could get a new release out please? :)
> 
> EDIT: preferably after #304 is merged too so that everyone gets the fix without the need to do `npm audit fix` or similar.

Yeah, that way packages that are dependent on this one like `postcss-sass` (so `stylelint` indirectly) can release updates with the vulnerability fixed.

**Comment by tonyganch** [comment]
Published as v4.3.0 🐙
